scripts/run_simulator.sh

```diff
@@ -31,12 +31,12 @@ run_simulator() {
     popd
     echo "running simulator from $PWD"
     ./cmd/simulator/simulator \
-        --endpoints=$RPC_ENDPOINTS \
+        "$@" \
```

Comment:
If we're going to use this, would it make sense to using only this instead of passing some of the parameters via `$@` and others set in the script?

Reply:
not opposed to this but I think it's cleaner as is.